lib/libutee/include/pta_system.h

```diff
@@ -189,4 +189,21 @@
  */
 #define PTA_SYSTEM_GET_TPM_EVENT_LOG	12
 
+/*
+ * Send an OCALL to the calling TA's Client Application
+ *
+ * Note that if TA 1 invokes TA 2, which invokes TA N+1, it is the CA of TA 1
+ * which will receive the OCALL request.
```

Comment:
A minor thing. While reviewing I started to wonder if this is really a good thing. I thought about possible conflicts with command IDs and CAs executing commands they are not supposed to execute as they come from a TA they have not invoked directly. But then I realized that the TA UUID is passed to the CA so the context is clear.

So, how about reminding the reader that in any case the OCALL requests contains the UUID of the originator TA?

Reply:
I have added a comment to that effect.